theme/migrations/0005_userquota.py

```diff
@@ -45,7 +45,7 @@ class Migration(migrations.Migration):
                 ('enforce_content_prepend', models.TextField(
                     default=b'Your action was refused because you have exceeded your quota. Your quota for HydroShare resources is {allocated}{unit} in {zone} zone. You currently have resources that consume {used}{unit}, {percent}% of your quota. ')),
                 ('content', models.TextField(
-                    default=b'To request additional quota, please contact support@hydroshare.org. We will try to accommodate reasonable requests for additional quota. If you have a large quota request you may need to contribute toward the costs of providing the additional space you need. See https://pages.hydroshare.org/about-hydroshare/policies/quota/ for more information about the quota policy.')),
+                    default=b'To request additional quota, please contact support@hydroshare.org. We will try to accommodate reasonable requests for additional quota. If you have a large quota request you may need to contribute toward the costs of providing the additional space you need. See https://help.hydroshare.org/about-hydroshare/policies/quota/ for more information about the quota policy.')),
```

Comment:
@cbcunc - What are the implications of changing a migration file that has already been migrated?
Does this need to be in the form of a new migration?

Reply:
There's no change to the model. This is simply an edit row contents. What are the implications I'm not aware of? We did this for xDCIShare without consequence.

Reply:
I assume this migration file won't be picked up by the migrate command. In that case the default value for the 'content' field would be the original value not the new one as edited in this migration file.

Reply:
Correct. @mjstealey already made the db content change through the admin interface manually, I believe. That's the kind of change we fix in two places.